### PR TITLE
Link Nostr account to existing user account.

### PR DESCRIPTION
**Pull request by ibz**

### CODE REVIEW

File: web/src/lib/components/settings/Nostr.svelte [hunk 1]
```diff
@@ -13,60 +76,70 @@
...
-        <div class="w-full flex items-center justify-center mt-4">
-            <div class="form-control w-full">
-                <input bind:value={nostr_public_key} type="text" id="nostr_public_key" name="nostr_public_key" class="input input-lg input-bordered" />
-            </div>
+        <div class="flex justify-center items-center mt-4 h-15">
+            {#if saveButtonActive}
+                <button id="save-profile" class="btn btn-primary" on:click|preventDefault={save}>Save</button>
+            {:else}
+                <button class="btn" disabled>Save</button>
+            {/if}
```
Comment (ktecho) [inline web/src/lib/components/settings/Nostr.svelte:108]: Maybe you can put this and avoid the `if`?

`class:btn-disabled={!saveButtonActive}`

File: web/src/lib/components/settings/Nostr.svelte [hunk 2]
```diff
@@ -13,60 +76,70 @@
...
-            <p>You're using your Plebeian Market generated Nostr identity. We created this for you as we detected
-                that you're not using a Nostr browser extension. This is not a problem and you can continue using
-                Plebeian Market this way, but it's highly recommended that you install a Nostr browser extension
-                and create your own Nostr identity so nobody else has access to your private key.</p>
-
-            <p class="mt-3 text-justify">You can use any of this browser extensions:
-                <a class="link" href="https://getalby.com/" target="_blank" rel="noreferrer">Alby</a>,
-                <a class="link" href="https://github.com/fiatjaf/nos2x" target="_blank" rel="noreferrer">nos2x</a> or
-                <a class="link" href="https://www.blockcore.net/wallet" target="_blank" rel="noreferrer">Blockcore</a>.
-            </p>
```
Comment (ktecho) [inline web/src/lib/components/settings/Nostr.svelte:44]: I think we must give some information on what Nostr is and how to get an extension. If not here, maybe in the FAQ.

File: web/src/lib/components/settings/Nostr.svelte (hunk 1 above)
Comment (ibz) [reply web/src/lib/components/settings/Nostr.svelte:108]: Unfortunately not, "disabled" is not a class, but an attribute :(

File: web/src/lib/components/settings/Nostr.svelte (hunk 2 above)
Comment (ibz) [reply web/src/lib/components/settings/Nostr.svelte:44]: Yep, sorry, I missed this. Added to the FAQ.

**Pull request merged**